ui/src/components/ui/device-statistics/device-statistics.tsx

```diff
@@ -34,7 +34,7 @@ export const DeviceStatistics = observer(() => {
       />
       <StatisticCard
         className={styles.deviceStatistics}
-        color='var(--vkui--color_accent_orange)'
+        color='#792EC0'
```

Comment:
Extract the color to a css variable
https://github.com/VKCOM/devicehub/blob/master/ui/src/styles/variables/global.css#L10